docs/source/changelog.rst

```diff
@@ -0,0 +1,55 @@
+Changelog
+##############################################################################
+
+This page lists updates to CS Unplugged.
+All notable changes to this project will be documented in this file.
+
+We base our numbering system from the guidelines at `Semantic Versioning 2.0.0`_,
+however since our project started before it was migrated to GitHub, the first
+major open source release is being labeled as ``2.0.0``.
+
+Given a version number MAJOR.MINOR.HOTFIX:
+
+- MAJOR version change when major text modifications are made (for example: new
+  topic).
+- MINOR version change when content or functionality is added or updated (for
+  example: new videos, new activities, large number of text (typo/grammar) fixes).
+- HOTFIX version change when bug hotfixes are made (for example: fixing a typo).
+- A pre-release version is denoted by appending a hyphen and the alpha label
+  followed by the pre-release version.
+
+We have listed major changes for each release below.
+
+
+2.0.0-alpha.1
+==============================================================================
+
+- **Release date:** 20th June 2017
+- **Downloads:** `Source downloads are available on GitHub`_
+
+**Notable changes:**
+
+The first major step in releasing a open source version of CS Unplugged.
+While some existing content from the classic version of CS Unplugged have yet
+to be adapted into the new format and system, we are releasing this version as
+a sneak peek for teachers.
+
+The following topics are available in this version:
+
+- Binary numbers:
+
+  - Two lessons for ages 5 to 7.
+  - Three lessons for ages 8 to 11.
+  - Seven curriculum integrations.
+
+- Error detetction and correction:
+
+  - Two lessons for ages 8 to 11.
+  - Five curriculum integrations.
+
+- Sorting networks:
+
+  - One lesson for ages 8 to 10.
+
```

Comment:
I think the changelog needs to include backend changes - particularly as devs are far more likely to read this than authors.